examples/neural_networks/plot_random_nn_overfitting.py

```diff
@@ -0,0 +1,71 @@
+"""
+===========================================================================
+Impact of increasing the number of hidden neurons in random neural networks
```

Comment:
Yah, I think it's better to merge the two examples together  into one example with a global title "Effect of parameters in RandomBasisFunction"